Automattic/wp-calypso
# A4A: Sites Dashboard - Fix Dataviews keeping sites in an error state on all pages

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/491

## Proposed Changes

* This PR fixes a problem with Dataviews not removing sites in error states from the table.

## Why are these changes being made?

* This is a bug.

## Testing Instructions

* To test this, you need to have enough sites to trigger pagination and at least one site in an error state
* Navigate through the pages
* Verify that sites with errors appear only on the correct page

## Pre-merge Checklist
